pipeline/interpolate_gene.py

```diff
@@ -63,6 +63,15 @@ def parse_args():
         Path of the interpolator checkpoints.
         """,
     )
+    parser.add_argument(
+        "--saving_format",
```

Comment:
I would do `--saving-format` to be consistent

Reply:
Good points! Addressed here: [d53c77d](https://github.com/BlueBrain/Deep-Atlas/pull/64/commits/d53c77d999e553cb479e5a8e14f1dd7786ba4954)